code/modules/mining/mine_items.dm

```diff
@@ -428,3 +431,21 @@
 	anchored = 1
 	layer = MOB_LAYER - 0.2
 	density = 0
+
+/obj/item/weapon/emptysandbag
+	name = "empty sandbag"
+	desc = "A bag to be filled with sand."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "sandbag"
+	w_class = 1
+
+/obj/item/weapon/emptysandbag/attackby(obj/item/W, mob/user, params)
+	if(istype(W,/obj/item/weapon/ore/glass || /obj/item/weapon/ore/glass/basalt))
```

Comment:
Im not even sure how this compiled... but this doesn't fucking work.

You would only need istype(W,/obj/item/weapon/ore/glass) since basalt is a child.